app/src/main/res/layout/my_downloads_fragment.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/my_downloads_app_bar_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:theme="@style/ThemeOverlay.AppCompat.ActionBar"
+      app:layout_constraintBottom_toTopOf="@+id/my_downloads_tabs_viewpager_container"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent">
+
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:background="@color/oppiaPrimaryDark"
+        android:textSize="20sp"
+        app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
+        app:title="@string/my_downloads"
+        app:titleTextColor="@color/white" />
+
+      <com.google.android.material.tabs.TabLayout
+        android:id="@+id/my_downloads_tabs_container"
+        style="@style/AppTabLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="14sp"
+        app:tabBackground="@color/oppiaPrimaryDark"
+        app:tabGravity="fill"
+        app:tabIconTint="@color/tab_icon_color_selector"
+        app:tabIndicatorColor="@android:color/white"
+        app:tabIndicatorHeight="2dp"
+        app:tabMode="fixed"
+        app:tabTextAppearance="@style/AppTabTextAppearance"
+        app:tabTextColor="@color/tab_icon_color_selector" />
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <FrameLayout
+      android:id="@+id/my_downloads_tabs_viewpager_container"
+      android:layout_width="match_parent"
+      android:layout_height="0dp"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/my_downloads_app_bar_layout">
+
+      <androidx.viewpager.widget.ViewPager
+        android:id="@+id/my_downloads_tabs_viewpager"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+      <View
+        android:id="@+id/toolbar_shadow_view"
```

Comment:
Nit: Rename the widgets as per the file name or please remove it if its not being used

Reply:
I have filed an issue for this because we need to fix this in other files too.
https://github.com/oppia/oppia-android/issues/556